src/common/dcsvg.cpp

```diff
@@ -681,22 +779,21 @@ void wxSVGFileDCImpl::DoDrawPolygon(int n, const wxPoint points[],
                                     wxPolygonFillMode fillStyle)
 {
     NewGraphicsIfNeeded();
-    wxString s;
 
-    s = wxS("  <polygon style=\"");
-    if (fillStyle == wxODDEVEN_RULE)
-        s += wxS("fill-rule:evenodd;");
-    else
-        s += wxS("fill-rule:nonzero;");
+    wxString s;
 
-    s += wxS("\" ") + wxGetPenPattern(m_pen) + wxGetBrushFill(m_brush) + wxS(" points=\"");
+    s = wxS("  <polygon points=\"");
 
     for (int i = 0; i < n; i++)
     {
         s += wxString::Format(wxS("%d %d "), points[i].x + xoffset, points[i].y + yoffset);
         CalcBoundingBox(points[i].x + xoffset, points[i].y + yoffset);
     }
-    s += wxS("\"/>\n");
+
+    s += wxString::Format(wxS("\" %s %s %s style=\"fill-rule:%s;\"/>\n"),
+        wxRenderMode(m_renderingMode), wxGetPenPattern(m_pen), wxGetBrushFill(m_brush),
+        (fillStyle == wxODDEVEN_RULE) ? wxS("evenodd") : wxS("nonzero"));
```

Comment:
```suggestion
        fillStyle == wxODDEVEN_RULE ? wxS("evenodd") : wxS("nonzero"));
```
IMHO extra parentheses hurt rather than help readability here.